dashboards/vmagent.json

```diff
@@ -3505,15 +3505,15 @@
               },
               "editorMode": "code",
               "exemplar": true,
-              "expr": "sum(vm_promscrape_targets{job=~\"$job\", instance=~\"$instance\", status=\"down\"}) by(job, type) > 0",
+              "expr": "sum(vm_promscrape_scrape_pool_targets{job=~\"$job\", instance=~\"$instance\", status=\"down\"}) by(job, scrape_job) > 0",
```

Comment:
Not sure if it is required to use new metric in the dashboard.
Currently, this panel allows to discover number of targets per discovery type and job name. Updated version does not show discovery type anymore and shows `job` / `scrape_job` combinations instead. Current version seems to be more useful to me, wdyt?

Users who need to track `job` / `scrape_job` can update the dashboard to match their needs, but it seems to me like this is rarely needed.

Reply:
Oh, I was thinking the very opposite [here](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5335#issuecomment-1815972290).
IMO, the `type` label that `vm_promscrape_targets` has is not very helpful. For example, if i saw `vm_promscrape_scrapers_stopped_total{type="consul_sd_configs"}` dropped from `100` to `0`, `vm_promscrape_targets` can't help me to find out the exact scrape_job-> scrape_config which has problems, I still need to check  /targets endpoints, right?
I thought to add `type` label to `vm_promscrape_scrape_pool_targets` as well, but it will increase the metrics number by 10+ times, and I don't think it's worth it.

>Current version seems to be more useful to me

I don't see how, please share some insights here)

Reply:
Good point. I was thinking about a bit different perspective, when there are separate teams operating VictoriaMetrics(platform team) and teams who deploy their applications to be monitored(developers).
In this case platform team members won't know job names and expectations on number of targets per specific job.  Moreover, in case there are a lot of jobs panel might become unreadable.

But in order to cover both cases we can have separate panels, one with generic view per discovery type(as it will contain less unique items and might be easier to read). Same panel will have a drilldown option there which will open a more detailed panel with per-job graph. What do you think?

Reply:
Cool, I will add new one instead of replacing.

Reply:
Added two panels, please check. @zekker6 
<img width="1404" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/39937150/b3143426-da26-49f2-80ec-486abe88cfef">